### PR TITLE
[Lagos-2022] initial event stub

### PR DESCRIPTION
**Pull request by haroldsphinx**
- CFP page updated

### CODE REVIEW

File: data/events/2022-lagos.yml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+# Leave CFP dates blank if you don't know yet, or set all three at once.
+cfp_date_start: August 1st, 2022 # start accepting talk proposals.
+cfp_date_end: October 3rd, 2022 # close your call for proposals.
+cfp_date_announce: August 1st, 2022 # inform proposers of status
+
+cfp_link: "https://sessionize.com/localhost-conference-2022" #if you have a custom link for submitting proposals, add it here. This will control the Propose menu item as well as the "Propose" button.
+
+registration_date_start: August 1st, 2022 # start accepting registration. Leave blank if registration is not open yet
+registration_date_end: October 28th, 2022 # close registration. Leave blank if registration is not open yet.
```
Comment (phrawzty) [inline data/events/2022-lagos.yml:25]: All dates must be in the format `YYYY-MM-DDTHH:MM:SS+TZ:TZ`, as is indicated in this exact file (see lines 9—11). This was feedback on your other PR (that you closed) also.

File: data/events/2022-lagos.yml [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    image: "adedayo.jpg"
+  - name: "Idowu Emehinola"
+    employer: "Advanced Blockchain"
+    twitter: "@_hydeenoble"
+    github: "hydeenoble"
+    image: "idowu.jpg"
+  - name: AbdulBasit Kabir
+    employer: "Interswitch"
+
+organizer_email: "ga" # Put your organizer email address here
```
Comment (phrawzty) [inline data/events/2022-lagos.yml:67]: Not sure what `ga` is but this must be your mailing list.

```suggestion
organizer_email: "lagos@devopsdays.org" # Put your organizer email address here
```

File: data/events/2022-lagos.yml [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+startdate: October 29th, 2022 # The start date of your event. Leave blank if you don't have a venue reserved yet.
+enddate: October 29th, 2022 # The end date of your event. Leave blank if you don't have a venue reserved yet.
+
+# Leave CFP dates blank if you don't know yet, or set all three at once.
+cfp_date_start: August 1st, 2022 # start accepting talk proposals.
+cfp_date_end: October 3rd, 2022 # close your call for proposals.
+cfp_date_announce: August 1st, 2022 # inform proposers of status
+
+cfp_link: "https://sessionize.com/localhost-conference-2022" #if you have a custom link for submitting proposals, add it here. This will control the Propose menu item as well as the "Propose" button.
```
Comment (phrawzty) [inline data/events/2022-lagos.yml:22]: The branding here makes it quite clear that this is an event called "Localhost".

image

DevOpsDays—the global brand—is _not_ "presenting" this event. If you intend to co-locate `DevOpsDays Lagos` alongside the Localhost event, that's fine, but you need to be extremely clear with the branding.

File: data/events/2022-lagos.yml [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+#
+location_address: "Plot 9 Gbagada Industrial Scheme, Beside UPS, Gbagada Expressway" #Optional - use the street address of your venue. This will show up on the welcome page if set.
+
+nav_elements: # List of pages you want to show up in the navigation of your page.
+ # - name: propose
+ # - name: location
+ # - name: registration
+ # - name: program
+ # - name: speakers
+  - name: sponsor
```
Comment (phrawzty) [inline data/events/2022-lagos.yml:43]: Keep this disabled until you're ready to accept sponsors (see also line 78).

```suggestion
 # - name: sponsor
```

**Review by phrawzty — CHANGES_REQUESTED** [review]

File: data/events/2022-lagos.yml (hunk 3 above)
Comment (haroldsphinx) [reply data/events/2022-lagos.yml:22]: Yes, working on updating the branding for this

File: data/events/2022-lagos.yml (hunk 4 above)
Comment (haroldsphinx) [reply data/events/2022-lagos.yml:43]: We are actually ready to accept sponsorships, updated L78

File: data/events/2022-lagos.yml [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+
+registration_date_start: "2022-08-01T10:00:00Z+01:00" # start accepting registration. Leave blank if registration is not open yet
+registration_date_end: "2022-10-28T00:00:00Z+01:00" # close registration. Leave blank if registration is not open yet.
+
+registration_closed: "" #set this to true if you need to manually close registration before your registration end date
+registration_link: "" # If you have a custom registration link, enter it here. This will control the Registration menu item as well as the "Register" button.
+
+# Location
+#
+coordinates: "6.54451°, 3.39226°" # The coordinates of your city. Get Latitude and Longitude of a Point: http://itouchmap.com/latlong.html
```
Comment (somatorio) [inline data/events/2022-lagos.yml:32]: You need to remove the "°", google maps (on location page) won't be able to pinpoint the location as is

File: data/events/2022-lagos.yml [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+description: "Devopsdays is coming to Lagos!" # Edit this to suit your preferences
+ga_tracking_id: "" # If you have your own Google Analytics tracking ID, enter it here. Example: "UA-74738648-1"
+cancel: "false"
+
+# All dates are in unquoted 2020-MM-DDTHH:MM:SS+TZ:TZ, like this:
+#   variable: 2019-01-04T00:00:00+02:00
+#   variable: 2019-01-05T23:59:59+02:00
+# Note: we allow 2020-MM-DD for backward compatibility, but it can lead to unexpected behaviors (like your event disappearing from the front page during your last day)
+
+startdate: "2022-10-29T10:00:0+01:00" # The start date of your event. Leave blank if you don't have a venue reserved yet.
```
Comment (phrawzty) [inline data/events/2022-lagos.yml:14]: The formatting here is really picky.

```suggestion
startdate: "2022-10-29T10:00:00+01:00" # The start date of your event. Leave blank if you don't have a venue reserved yet.
```

File: data/events/2022-lagos.yml (hunk 5 above)
Comment (phrawzty) [inline data/events/2022-lagos.yml:32]:
```suggestion
coordinates: "6.54451, 3.39226" # The coordinates of your city. Get Latitude and Longitude of a Point: http://itouchmap.com/latlong.html
```

**Review by phrawzty — CHANGES_REQUESTED** [review]
More changes requested.

File: data/events/2022-lagos.yml (hunk 3 above)
Comment (haroldsphinx) [reply data/events/2022-lagos.yml:22]: This has been resolved now

**Review by somatorio — APPROVED** [review]
LGTM, but I want to know @phrawzty opinion on the branding part fix

**Comment by phrawzty** [comment]
@haroldsphinx,

With all due respect, this is very clearly advertising an event called "Localhost Conference 2022", as proposed by an organisation of the same name. There's nothing here that indicates that it's a DevOpsDays event except some light branding on the ticket sales page.

At this point I am disinclined to accept this PR or sign off on the local chapter.

If you wish to continue, I encourage you and your local team to (re-)read the [organising page](https://devopsdays.org/organizing), and to decide if you're actually running a DevOpsDays event or not. From there, I invite you to send an email to `info@devopsdays.org` with the following details:
- The exact material relationship between Localhost and the proposed DevOpsDays Lagos event
- Proposed / planned schedule for both events
- Strategy for running the mandatory open spaces

To be clear, I am _not_ discouraging you from organising and running a local event. Our responsibility as Core Organisers is to protect the brand, shepherd local events, and curate a fundamental experience. Our goal, therefore, is to help you do just that, if you so choose.

Please take the time to consider this all carefully before acting. If you choose not to associate with the DevOpsDays brand, that's OK—no hard feelings, and I wish you the best for your future events.

If you do want to run a DevOpsDays event, then we'll be happy to engage with you to ensure that it's done correctly. My final piece of advice should you wish to go down this path: I strongly encourage you to consider running a _single, independant DevOpsDays event—unrelated to Localhost_—at least for the first time…

**Comment by phrawzty** [comment]
This PR has been open for a month with no activity. Let's give it a few more days, and if there's no activity, we'll go ahead and close it.

**Comment by phrawzty** [comment]
Closing this PR due to inactivity.

**Pull request closed without merging**